.github/workflows/lint_test.yml

```diff
@@ -43,11 +43,25 @@ jobs:
     # Even thought we have readthedocs build docs on every MR, this is useful
     # because we build with -W --keep-going (see Makefile) and this will
     # therefore fail on warnings that could be ignored by readthedocs and
-    # lead to half-broken docs
+    # lead to half-broken docsXXX
     - name: build docs
       # This should make the version in .readthedocs.yaml in the repository root
       if: matrix.python-version == '3.12'
       run: |
         cd docs
         python -m pip install -r requirements.txt
         make html
+  check-commit-message:
+    name: Check Commit Message
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Commit Type
+        uses: gsactions/commit-message-checker@v2
```

Comment:
This repo feels not super maintained (last commit 3 years ago). Wondering if we really need an external dependency to run a regexp against commit messages ?

Like it seems those would be available [in variables](https://stackoverflow.com/a/67145320) and maybe its just a matter of writing a small python script to iterate on each and run the regexp ?